### ISSUE TITLE
Stocks APIs - Multiple/Array Tickers in requests and  responses

### ISSUE THREAD

**Issue by papadeus** [polygon-io/feature-requests#7]
It is frustrating when we need to make the same call repeatedly for different tickers. For example, if we want aggregates for the same period and options but for 1,000 tickers, it requires 1,000 API calls. It would be much nicer if an array of tickers could be passed in and an array of results returned. This applies to ALL stock endpoints.

Varying tickers are the biggest frustration but there are times when other fields could also be arrays, for example the timestamp of Quotes NBBO VX (I want last quote for a given ticker at an array of timestamps)

Example:
POST:    /v2/aggs/tickers
BODY:
{
    "tickers": ["MSFT","AAPL","TSLA"],
    "multiplier": 2,
    "from": "2021-01-02",
    "to": "2021-01-03"
}

OR (this would have query length limitations but would still be an improvement)
GET:    /v2/aggs/tickers/range/{multiplier}/{timespan}/{from}/{to}?tickers=AAPL,TSLA,MSFT,...

**Comment by RaccoonWang** [comment]
+1, would like to see this implemented ASAP

**Comment by Diogomartf** [comment]
Yes, I need this, is it implemented already? Are plans to add this in the future?
